src/ManifoldMuseum.jl

```diff
@@ -181,7 +241,59 @@ function log(M::Manifold, x, y)
     return v
 end
 
-manifold_dimension(M::Manifold) = error("manifold_dimension not implemented for a $(typeof(M)).")
+"""
+    geodesic(M::Manifold, x, v)
+
+Get the geodesic with initial point `x` and velocity `v`. The geodesic
+is the curve of constant velocity that is locally distance-minimizing. This
+function returns a function of time, which may be a `Real` or an
+`AbstractVector`.
+"""
+geodesic(M::Manifold, x, v) = t -> exp(M, x, v, t)
+
+"""
+    geodesic(M::Manifold, x, y, t)
+
+Get the point at time `t` traveling from `x` along the geodesic with initial
+point `x` and velocity `v`.
+"""
+geodesic(M::Manifold, x, v, t) = exp(M, x, v, t)
+
+"""
+    geodesic(M::Manifold, x, y, T::AbstractVector)
```

Comment:
There's a typo: `y` should be `v`. Whatever you decide above, obviously, `geodesic` must be consistent regarding the `T`-vector.